extensions/data-transfer/portability-data-transfer-daybook/src/main/java/org/datatransferproject/transfer/daybook/social/DaybookPostsImporter.java

```diff
@@ -155,6 +164,7 @@ private String insertActivity(SocialActivityModel activity, TokensAndUrlAuthData
 
     FormBody formBody = builder.build();
     requestBuilder.post(formBody);
+    size = formBody.contentLength();
```

Comment:
Ok but are we sure we want to log the payload size here? 
E.g. what if the Daybook unpacks it / some fields in the form are stored more efficiently?

I'd say let's merge this and then check the numbers for real transfers..